core-js/src/test/javascript/cdf/components/UnmanagedComponent-spec.js

```diff
@@ -219,10 +222,85 @@ define([
     });
 
     /**
+     * ## Unmanaged Component # should allow changes in the datasources
+     */
+    it("should allow changes in the datasources", function(done) {
```

Comment:
Use a more generic description. E.g.: "preExecution should allow updating the query definition object"